Podfile

```diff
@@ -1,24 +1,32 @@
 platform :osx, '10.9'
 use_frameworks!
 
-def common_pods
-  pod 'PathKit', '~> 0.8.0', inhibit_warnings: true
+target 'swiftgen' do
+  pod 'Commander', '~> 0.8'
   pod 'StencilSwiftKit', '~> 2.3'
-  pod 'SwiftGenKit', '~> 2.1'
+  podspec :path => 'SwiftGenKit.podspec'
   pod 'Yams', '~> 0.3'
-end
 
-target 'swiftgen' do
-  common_pods
-  pod 'Commander', '~> 0.8'
+  target 'SwiftGen UnitTests' do
+    inherit! :complete
+  end
+
+  target 'Templates UnitTests' do
+    inherit! :complete
```

Comment:
Did you try with simply `inherit! :search_path` instead of `:complete`?

Reply:
Yeah, it's not enough it seems, because it starts complaining about `no such module StencilSwiftKit`.